DummyUserContextManager.py

```diff
@@ -0,0 +1,12 @@
+from AbstractUserContextManager import AbstractUserContextManager
+from random import *
+
+class DummyUserContextManager(AbstractUserContextManager):
+    def __init__(self):
+        pass
+
+    def get_user_and_contexts(self):
+        return [1], [1, 2, 3]
```

Comment:
We should update this to match the return format used by the other UserContextManagers -- namely:
`return 1, numpy.ndarray([1, 2, 3])`